internal/pkg/manifest/lb_web_svc.go

```diff
@@ -166,7 +168,15 @@ func (t *TaskConfig) TaskPlatform() (*string, error) {
 	if t.Platform == nil {
 		return nil, nil
 	}
-	return t.Platform.PlatformString, nil
+	return aws.String(dockerengine.PlatformString(t.Platform.OS(), t.Platform.Arch())), nil
+}
+
+// IsWindows returns whether or not the service is building with a Windows OS.
+func (t TaskConfig) IsWindows() bool {
```

Comment:
Can we make this one private 🤔

Reply:
We now use it in the template package, so I changed it to public.